PKGBUILD

```diff
@@ -35,7 +35,7 @@ build() {
     yarn --non-interactive --frozen-lockfile
     NODE_ENV=production yarn --non-interactive --frozen-lockfile build
   )
-  go run github.com/gobuffalo/packr/packr
+  go run hack/packr/packr.go
   local build_date=$(date "+%F-%T" -d "@${SOURCE_DATE_EPOCH}")
   go build \
```

Comment:
add
```
export CGO_LDFLAGS="$LDFLAGS"
export CGO_CFLAGS="$CFLAGS"
```
and drop the `-extldflags`.

No `GOFLAGS` because it fails for go test for some reason.

Reply:
These were also used in the guide you linked. I'll add them too.
```
  export CGO_CPPFLAGS="${CPPFLAGS}"
  export CGO_CXXFLAGS="${CXXFLAGS}"
```

Reply:
Ok, but they probably don't do anything.

> The other changes is explicitly listing up the needed CGO_* variables needed.
> Levente pointed out CGO_CFLAGS is still needed for FORTIFY_SOURCE, so for the
> sake of completeness it's all listed.

https://lists.archlinux.org/pipermail/arch-dev-public/2020-April/029944.html

Reply:
Alr8, I'll remove them :D.